src/app/page.tsx

```diff
@@ -7,6 +7,16 @@ export default function Home() {
         <Alert type="success" headingLevel="h1">
           App running successfully
         </Alert>
+        <h1>Welcome</h1>
+        <h1>Welcome</h1>
+        <h1>Welcome</h1>
+        <h1>Welcome</h1>
+        <h1>Welcome</h1>
+        <h1>Welcome</h1>
+        <h1>Welcome</h1>
+        <h1>Welcome</h1>
+        <h1>Welcome</h1>
+        <h1>Welcome</h1>
```

Comment:
I think this causes an accessibility issue?

<img width="1188" alt="Screenshot 2024-03-12 at 11 53 40 AM" src="https://github.com/coforma/swift-tech-challenge/assets/57802560/c85125a7-0a49-4811-a64f-2eade3caa14e">

Reply:
Funnily enough, thats ok. Though yeah, we should reduce it to 1. It's mad that theres a role='navigation' on a div in the header. Thats not a unique landmark so its through an error. That being said, we haven't included navigation in the header yet so removing the role altogether is the best approach atm.